README.md

```diff
@@ -205,3 +205,8 @@ on any directory that contains `tf` files with the following criteria
 3. A directory with symlinked files that are linked to files that changed
 4. A directory that that uses a Terraform module whose source changed
 5. A directory with Terraform files that refer to an autovar file that changed
+
+### backend_check
+
+`backend_check [directory]` verifies that all directories under the given directory that contain `.tf` files
```

Comment:
We used `<directory>` in other places in this file. Probably should be consistent

Reply:
`<>` usually denotes a required argument. `[]` is usually used for optional arguments. So I think what I wrote is accurate.